### ISSUE TITLE
Monitor suddenly keeps freezing

### ISSUE THREAD

**Issue by ayapp99** [max99x/pagemon-chrome-ext#362]
I have used PM over almost two year; it worked well.  However, it suddenly stopped to remind me for web changes. No any responds to any changes.  I tried to reinstall my win 7, chrome (clear call personal data)  and PM, which were useless. It's still keep silence. 

My computer is Windows 7 Home Premium.
Chrome version is 43.0.2357.130 m.

I searched some old posts and tried to do this step  (Wrench -> 
Tools -> Extensions -> Developer Mode -> expand Page Monitor -> 
"background.htm" -> Console), it pinpointed this message-----Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check http://xhr.spec.whatwg.org/.

Please help me

Thank you

**Comment by ayapp99** [comment]
i tried to monitor this page and had never got any remind.

http://www.neimanmarcus.com/Vince-Short-Sleeve-Contrast-Trim-Blouse/prod178490622_cat51010798__/p.prod?icid=&searchType=EndecaDrivenCat&rte=%252Fcategory.service%253FitemId%253Dcat51010798%2526pageSize%253D120%2526No%253D0%2526Ns%253DMAX_PROMO_PRICE%2526refinements%253D4294946602%252C574%252C614&eItemId=prod177170226&cmCat=product

I noticed a "null" variable set when exporting bookmarks.html.

<DT><A HREF="http://shop.nordstrom.com/s/miu-miu-pointy-toe-pump-women/3911297?origin=category&BaseUrl=Women" ADD_DATE="1435806981885">Miu Miu Pointy Toe Pump (Women) | Nordstrom</A>



Thank you.

**Comment by max99x** [comment]
I've tried monitoring that page and got a notification today. It wasn't very meaningful, as they just removed a link, but it did trigger. That console error is harmless and should not be causing any issues. The export record looks fine too - the nulls just mean that you haven't set any custom monitoring settings for that specific page. The fact that last_check and last_changed are the same suggests that PM thinks the page is currently marked as changed (since Wed Jul 01 2015 20:22:26 GMT-0700). Does the Page Monitor icon in the toolbar not have a green number badge?

**Comment by ayapp99** [comment]
No, PM icon is very clean without any green number badges. Even  I know the page is changed, my PM is still silent.  My Win7 system,chrome and PM are all fresh re-installed, I am confused and don't know how to deal with the problem.

**Comment by max99x** [comment]
On the options page, when does it say the last check was, and does anything happen when you click the  little refresh button next to the Last Checked label of the page?

**Comment by ayapp99** [comment]
The last check is keeping changing very slowly but without numerical order sometimes. 

If I click the little refresh button, it return to 0 sec before, then 5 sec beore, 6 sec before, 8 sec before, 9 sec before, 10 sec before, 12sec before, 13 sec before, 11 sec before, 7 sec before, 10 sec before, 11sec before, 0sec before, 1sec before, 0 sec before, 2 sec before, 3 sec before, 4 sec before, 5 sec before, 6 sec before, 5 sec before, 7 sec before, 8 sec before, 10 sec before.   13 sec before,   with a very slow speed and no remind.

Thank you.

**Comment by max99x** [comment]
Is this happening to one page or to all? If you can export your settings/pagelist and send it to me at max99x@gmail.com, I can try to debug this further.

**Comment by ayapp99** [comment]
It happened to almost all pages. I did export pagelist and sent to your email. Thank you so much.
